packages/Pagination/Pagination.md

```diff
@@ -4,7 +4,19 @@
 - The component can take any content as children inside a Panel
 - Any number of panels can be used to paginate
 - Style your content within Panel however you wish
-- copy can be passed in as a prop, en is set as default
+
+### Accessibility
+
+- You must pass in a value to `copy` in order to render helpful text to assistive technology users
+
+```jsx
+<Pagination copy="en">
+  <Pagination.Panel>Content 1: any type of content can be used here.</Pagination.Panel>
+  <Pagination.Panel>Content 2: use responsibly.</Pagination.Panel>
+  <Pagination.Panel>Content 3: Pagination Panels take &lt;div /&gt; tags, image tags, heading tags and so on.</Pagination.Panel>
+</Pagination>
+<Box inset={4}> </Box>
+```
```

Comment:
Do we need to show two similar code examples? Existing example also demonstrates a `copy` prop.

Reply:
Agreed, this can probably be removed.